apps/client/src/config/matomo.js

```diff
@@ -15,19 +15,37 @@ export const matomo = {
 
 export const trackingEnabled = () => !localStorage.getItem("doNotTrack");
 
-export const categories = {
-  navigate: "navigeren",
+export const actions = {
+  CLICK_EXTERNAL_NAVIGATION: "externe navigatie",
+  CLICK_INTERNAL_NAVIGATION: "interne navigatie",
+  CLICK_PHONE_LINK: "telefoonnummer",
+  CONCLUSION: "tonen conclusie",
+  DOWNLOAD: "download",
+  EDIT_ADDRESS: "wijzig adres",
+  EDIT_QUESTION: "wijzig vraag",
 };
 
-export const actions = {
-  clickExternalLink: "Externe link",
-  clickPhoneLink: "Telefoonnummer",
+export const sections = {
+  CONCLUSION: "conclusie",
+  CONTACT_CONCLUSION: "contact conclusie",
+  INTRO: "intro",
+  LOCATION_RESULT: "locatie resultaat",
 };
 
 export const eventNames = {
-  ADDRESS_ERROR: "Address error",
-  ADDRESS_NOT_FOUND: "Address not found",
-  FOOTER: "Footer",
-  INTRO: "Intro",
-  INTRO_EXTRAORDINARY: "Intro - Extraordinary situation",
+  ADDRESS_ERROR: "adres error",
+  ADDRESS_NOT_FOUND: "adres niet gevonden",
+  APPLY_FOR_PERMIT: "vergunning aanvragen",
+  BACK: "terug naar",
+  DESCRIPTION: "toelichting",
+  FOOTER: "footer",
+  FORWARD: "naar de",
+  INTRO: "intro",
+  INTRO_EXTRAORDINARY: "extraordinary situation",
```

Comment:
`INTRO_EXTRAORDINARY` is unused (and untranslated).

Please replace all `<ContactSentence eventName={eventNames.INTRO_EXTRAORDINARY} />` by `<ContactSentence eventLocation="..." />`